spec/ttml3.xml

```diff
@@ -2499,8 +2549,7 @@ namespace names are listed in <specref ref="namespace-vocab-table"/>.
 <div2 id="ttml-content-doctype">
 <head>TTML Content Document Type</head>
 <p>The TTML Content Document Type is an <loc href="#terms-abstract-document-type">abstract document type</loc> of a profile
-of the Timed Text Markup Language intended to be used for interchange among distribution systems. This document type
-is defined in terms of the element and attribute vocabulary specified in <specref ref="vocabulary"/>.</p>
+of the Timed Text Markup Language defined in terms of a collection of functional <loc href="#terms-module">modules</loc>.</p>
```

Comment:
This statement presupposes that all parts of the specification are in a functional module, but it is not clear that that is the case. For just one example, the conformance criteria are not in a module.

Reply:
No it doesn't. It says "The TTML Content Document Type is ...". Please read the entire paragraph and the older language in particular. It says exactly what needs to be said (at this point in the document).

Reply:
I did read it, and now I've read it again. I think you're saying that the abstract document type consists of entities which are defined by modules, which works, but the phrasing "defined in terms of" seems unclear to me. How about changing:

> defined in terms of a collection of functional modules

to

> whose content is defined by a collection of functional modules

?

Reply:
The editor prefers the current language "defined in terms of", which is clear English.

Reply:
I find myself disagreeing with the Editor on this point of clarity.